eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/list/flexibleadapter/FlexibleAdapterNotifier.java

```diff
@@ -42,14 +42,17 @@ public FlexibleAdapterNotifier(FlexibleAdapter flexibleAdapter)
     @Override
     public void notifyInitialItemsAdded(List initialItems)
     {
-        //noinspection unchecked
-        mFlexibleAdapter.addItems(0, initialItems);
+        notifyNewItemsAdded(initialItems, 0);
     }
 
 
     @Override
     public void notifyNewItemsAdded(List newItems, int positionStart)
     {
+        if (newItems == null || newItems.isEmpty())
```

Comment:
Right, it was the constructor that was called with null before, but this method should not be called  with null. Removed the null check.